pg_timetable.yaml

```diff
@@ -0,0 +1,173 @@
+package:
+  name: pg_timetable
+  version: "5.11.0"
+  epoch: 0
+  description: Advanced scheduling for PostgreSQL
+  copyright:
+    - license: PostgreSQL
+  dependencies:
+    runtime:
+      - curl
+      - postgresql-client
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 5046c1ad93f6588c330dcbf6b85d738a107caf42
+      repository: https://github.com/cybertec-postgresql/pg_timetable.git
+      tag: v${{package.version}}
+
+  - uses: go/bump
+    with:
+      deps: |-
+        golang.org/x/crypto@v0.35.0
+
+  - uses: go/build
+    with:
+      packages: .
+      ldflags: |
+        -X main.version=$(git describe --tags --abbrev=0)
+        -X main.commit=$(git show -s --format=%H HEAD)
+        -X main.date=$(git show -s --format=%cI HEAD)
+      output: pg_timetable
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: "Compatibility package to place binaries in the location expected by pg_timetable"
+    dependencies:
+      runtime:
+        - ${{package.name}}
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}
+          ln -sf /usr/bin/pg_timetable ${{targets.contextdir}}/pg_timetable
+    test:
+      pipeline:
+        - runs: "[ \"$(stat -c '%N' /pg_timetable 2>/dev/null)\" = \"'/pg_timetable' -> '/usr/bin/pg_timetable'\" ]"
+
+update:
+  enabled: true
+  github:
+    identifier: cybertec-postgresql/pg_timetable
+    strip-prefix: v
+
+test:
+  environment:
+    contents:
+      packages:
+        - postgresql
+        - build-base
+        - shadow
+        - sudo-rs
+        - glibc-locales
+    environment:
+      PGDATA: /tmp/test_db
+      PGUSER: wolfi
+      PGHOST: /tmp
+  pipeline:
```

Comment:
Can we have simple `--version | grep ${{package.version}}` here to test if those LDFLAGS are set properly?

Reply:
done bro, thanks